server/core/src/irods_signal.cpp

```diff
@@ -80,14 +80,20 @@ namespace irods
 
     const std::string STACKTRACE_DIR(irods::get_irods_stacktrace_directory().string());
 
-    void set_unrecoverable_signal_handlers() // TODO Consider calling this for grandpa also.
+    void set_unrecoverable_signal_handlers() // TODO Consider calling this for grandpa also. <<<
     {
-        rodsLog(LOG_DEBUG, "Setting stacktrace dump signal handler for process %d", getpid());
-        rodsLog(LOG_DEBUG, "Stacktraces will be dumped to \"%s\"", irods::STACKTRACE_DIR.c_str());
-        struct sigaction action{};
+        using log_server = irods::experimental::log::server;
+
+        log_server::debug("Setting stacktrace dump signal handler for process [{}].", getpid());
+        log_server::debug("Stacktraces will be dumped to [{}].", irods::STACKTRACE_DIR);
+
+        struct sigaction action;
+        action.sa_flags = 0;
+        sigemptyset(&action.sa_mask);
```

Comment:
@SwooshyCueb Did you leave `action.sa_mask` untouched on purpose?

I wasn't sure if there was a reason for not calling `sigemptyset` or `sigfillset` on it.

Reply:
I have no memory of this, so probably not

Reply:
Marking as resolved since we can infer from the original code that no signals should be blocked due to how `action` was originally initialized.